core/jazz_logs/test/test.js

```diff
@@ -234,6 +235,26 @@ describe('platform_logs', function() {
     assert.isTrue(acceptCount == 2);
   });
 
+  /*
+  * Given event.body.asset_type that is listed as valid in config, handler() should not inform of asset_type exception
```

Comment:
update comment and test name to conform to what is being tested

Reply:
Updated